### DISCUSSION TITLE
In-Network Rates are Percentages of Medicare Rates

### DISCUSSION THREAD

**Discussion by caracolina** [CMSgov/price-transparency-guide#515] [Q&A] [answered]
I have a question regarding the above scenario. We have a plan with many providers whose rates are based on a percentage of Medicare fee schedules. Since the negotiated rates fields have to be dollar amounts, does that mean we have to calculate rates for every procedure in CMS fee schedules for every participating provider?
Thanks in advance for any insights.

**Comment by shaselton-usds** [comment, marked as answer]
Rates based on a percentage of Medicare need to be calculated to a dollar amount.

**Reply by caracolina** [reply]
Thank you for your reply. My question is, do we have to publish every single rate conceivable or can we narrow it down? A pediatrician would be unlikely to provide OB/GYN services, etc.

**Reply by shaselton-usds** [reply]
Disclosure is required for anything that is included in the contract.
